Directory.Packages.props

```diff
@@ -7,7 +7,7 @@
     <PackageVersion Include="BenchmarkDotNet" Version="0.14.0" />
     <PackageVersion Include="BenchmarkDotNet.Diagnostics.Windows" Version="0.13.12" />
     <PackageVersion Include="CommandLineParser" Version="2.9.1" />
-    <PackageVersion Include="Newtonsoft.Json" Version="13.0.3" />
+    <PackageVersion Include="JsonPath.Net" Version="1.1.6" />
```

Comment:
### This file just determines the versions of all the packages referenced by any project in the solution, it is only referenced by GarnetJSON and it will not be included in the release binaries...